Client/Circle.h

```diff
@@ -4,15 +4,15 @@
 #include "Shape.h"
 #include "Vector2D.h"
 
+using namespace std;
+
 /**	
 	@brief Représente un cercle.
 
 	Un Circle est une Shape particulière défini par un rayon.
 */
 class Circle : public Shape {
 private:
-	/** @brief Encapsulation de la variable protected verticies. */
-	Shape::vertices;
```

Comment:
TL;DR Enhanced security.

This line, in case you don't know, is just switching the scope of the vertices variable from protected to private.
As any circular shape cannot be defined by a collection of vertices, (but only by a center point and a radius.) the variable is set to private to avoid it to be used in the Circle class or any derived one.